remme/rest_api/certificate.py

```diff
@@ -65,13 +68,28 @@ def put(cert, key, key_export):
     rem_sig = certificate_client.sign_text(crt_hash)
     crt_sig = get_certificate_signature(key, rem_sig)
 
+    save_p12(cert, key, path_to_save)
     status, _ = certificate_client.store_certificate(crt_bin, rem_sig, crt_sig.hex())
 
     return {'certificate': crt_export.decode('utf-8'),
             'priv_key': key_export.decode('utf-8'),
             'batch_id': re.search(r'id=([0-9a-f]+)', status['link']).group(1)}
 
 
+def save_p12(cert, private, path):
```

Comment:
Shouldn't we return binary from a separate endpoint instead?

Reply:
Yep. Sounds like better way